l1-contracts/deploy-scripts/DeployL1CoreUtils.s.sol

```diff
@@ -0,0 +1,296 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+// solhint-disable no-console, gas-custom-errors
+
+import {console2 as console} from "forge-std/Script.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+import {L1Bridgehub} from "contracts/bridgehub/L1Bridgehub.sol";
+import {DeployUtils} from "./DeployUtils.sol";
+import {L1Nullifier} from "contracts/bridge/L1Nullifier.sol";
+import {L1AssetRouter} from "contracts/bridge/asset-router/L1AssetRouter.sol";
+import {Governance} from "contracts/governance/Governance.sol";
+import {CTMDeploymentTracker} from "contracts/bridgehub/CTMDeploymentTracker.sol";
+import {ChainAdmin} from "contracts/governance/ChainAdmin.sol";
+import {L1NullifierDev} from "contracts/dev-contracts/L1NullifierDev.sol";
+import {L1ERC20Bridge} from "contracts/bridge/L1ERC20Bridge.sol";
+import {ChainAdminOwnable} from "contracts/governance/ChainAdminOwnable.sol";
+import {L1MessageRoot} from "contracts/bridgehub/L1MessageRoot.sol";
+import {L1ChainAssetHandler} from "contracts/bridgehub/L1ChainAssetHandler.sol";
+import {L1NativeTokenVault} from "contracts/bridge/ntv/L1NativeTokenVault.sol";
+import {BridgedStandardERC20} from "contracts/bridge/BridgedStandardERC20.sol";
+import {UpgradeableBeacon} from "@openzeppelin/contracts-v4/proxy/beacon/UpgradeableBeacon.sol";
+import {ProxyAdmin} from "@openzeppelin/contracts-v4/proxy/transparent/ProxyAdmin.sol";
+import {ContractsBytecodesLib} from "./ContractsBytecodesLib.sol";
+
+// solhint-disable-next-line gas-struct-packing
+struct DeployedAddresses {
```

Comment:
Do you think it makes sense to put all of these structs into a single file? Currently there is a bit of duplication I think, and its ok to keep these separate. They can be imported here, and reimported into other files.